manifests/charts/base/templates/clusterrole.yaml

```diff
@@ -115,7 +117,7 @@ rules:
   - apiGroups: ["authentication.k8s.io"]
     resources: ["tokenreviews"]
     verbs: ["create"]
-{{- if .Values.global.centralIstiod }}
+{{- if .Values.global.externalIstiod }}
```

Comment:
I don't think we can make backwards incompatible changes like this, it will break all users on 1.7

Reply:
should we support both for backwards compatible?

Reply:
Didn't we publicly announce it through every means possible? Release notes, blogs, etc? It seems like we probably don't want to just break this

Reply:
we call it external control plane in these blog posts, eg: `https://istio.io/latest/blog/2020/new-deployment-model/ ` so this is the reason I want to rename it as `externalIstiod`.